src/client/components/App/App.tsx

```diff
@@ -1,38 +1,61 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector, Provider } from 'react-redux';
 import { HistoryRouter as Router } from 'redux-first-history/rr6';
 import { Route, Routes } from 'react-router-dom';
 import styled from 'styled-components';
 import 'react-popper-tooltip/dist/styles.css';
 
-import { makeSampleDeck1 } from '@/factories/deck';
 import { isUserInitialized } from '@/client/redux/selectors';
-import { history, RootState } from '@/client/redux/store';
-
-import { DeckList } from '../DeckList';
-import { IntroScreen } from '../IntroScreen';
-import { Rooms } from '../Rooms';
-import { GameDisplay } from '../GameDisplay';
-import { GameManager } from '../GameManager';
+import { history, RootState, store } from '@/client/redux/store';
 
+const IntroScreen = React.lazy(() =>
+    import('../IntroScreen').then((module) => ({
+        default: module.IntroScreen,
+    }))
+);
+const Rooms = React.lazy(() =>
+    import('../Rooms').then((module) => ({
+        default: module.Rooms,
+    }))
+);
+const WebSocketProvider = React.lazy(() =>
+    import('../WebSockets').then((module) => ({
+        default: module.WebSocketProvider,
+    }))
+);
+const GameManager = React.lazy(() =>
+    import('../GameManager').then((module) => ({ default: module.GameManager }))
+);
+const GameDisplay = React.lazy(() =>
+    import('../GameDisplay').then((module) => ({ default: module.GameDisplay }))
+);
```

Comment:
Every item needs to be lazy-loaded in order to reduce bundle size.  

React's lazy loading feature checks to see if it truly can be lazy loaded before bundling it separately.  If we for instance, only try to lazy-load the GameDisplay + the websockets provider, b/c the websockets provider component is actually used by the Rooms + IntroScreen components, it will not be lazy-loaded / bundled away from `client.bundle.js`